6_drivers_prep.yml

```diff
@@ -4,63 +4,74 @@ packages:
   - sp
   - scipiper
   - dplyr
+  - ncdf4
+  - stringr
 
 sources:
   - 6_drivers/src/nldas_cube_utils.R
   - 6_drivers/src/nldas_feather_utils.R
-  
+
 targets:
-  6_drivers_prep: 
-    depends: 
+  6_drivers_prep:
```

Comment:
Is _prep intended to mean fetch+munge? I thought initially you had it meaning "prepare to fetch", but I'm pretty sure this file does a lot more than that now, right?

Reply:
no, the names are off again. 6 now includes all of the fetch, so I need to rename before I start a major data pull